src/sentry/api/endpoints/organization_plugins_configs.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+import six
+
+from sentry.plugins.base import plugins
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.plugin import PluginSerializer
+from sentry.models import ProjectOption, Project
+
+
+"""
+List one or more plugin configurations, including a `projectList` for each plugin which contains
+all the projects that have that specific plugin both configured and enabled.
+
+- similar to the `OrganizationPluginsEndpoint`, and can eventually replace it
+
+:qparam plugins array[string]: an optional list of plugin ids (slugs) if you want specific plugins.
+                               If not set, will return configurations for all plugins.
+"""
+
+
+class OrganizationPluginsConfigsEndpoint(OrganizationEndpoint):
+    def get(self, request, organization):
+
```

Comment:
the docstrings usually go right under the request method just fyi

Reply:
whoops... :)